app-server/app/src/main/resources/logback.xml

```diff
@@ -0,0 +1,13 @@
+<configuration>
+    <appender name="file" class="ch.qos.logback.core.FileAppender">
+        <file>logging-app-server.log</file>
```

Comment:
Logs should be going to `stdout` -- in a docker environment we'll be using the docker daemon to forward logs to some central location